src/cljs/main/broadfcui/utils/ajax.cljs

```diff
@@ -12,6 +12,8 @@
 ;; TODO - make this unnecessary
 (def content-type=json {"Content-Type" "application/json"})
 
+(def app-id {"X-App-Id" "FireCloud"})
+
```

Comment:
I noticed that the header keys for the various services in [firecloud-develop/1307](https://github.com/broadinstitute/firecloud-develop/pull/1307) were either all lower case or kabob/camel case. Is it safe to assume the header is not case sensitive?

Reply:
pretty much everything on the internet - plus empirical testing - says these comparisons are not case-sensitive.